editor/src/components/canvas/ui-jsx-canvas.tsx

```diff
@@ -431,7 +436,11 @@ export const UiJsxCanvas = betterReactMemo(
       })
 
       return (
-        <>
+        <div
+          style={{
+            all: 'initial',
+          }}
+        >
```

Comment:
This change was deliberate, and is the cause of the snapshot changes. Tailwind provides a ["preflight"](https://tailwindcss.com/docs/preflight) which adds CSS rules for the root of the doc. We obviously don't want that to affect the editor, so we scope all of the rules so that they are applied to descendants of the `canvas-container` only. That means we have to update those root rules to apply directly to the `canvas-container` element, which this `all: 'initial'` was interfering with. By moving the `all: 'initial'` up a level, we are still able to reset the styling on the canvas in a way that doesn't interfere with the tailwind preflight.